src/actions/expressions.js

```diff
@@ -7,7 +7,7 @@ import {
   getSelectedFrameId,
   getSource
 } from "../selectors";
-import { PROMISE } from "../utils/redux/middleware/promise";
+import { PROMISE } from "./utils/promise";
```

Comment:
hmm, i still like having the middleware directory so it is clear what it is:

`"./utils/middleware/promise";`

i agree redux is redundant